.gitpod.yml

```diff
@@ -3,10 +3,3 @@ image:
 
 tasks:
   - init: sudo wget -qO /usr/local/bin/yq https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 && sudo chmod a+x /usr/local/bin/yq && ./02_custom_docker_file_connecting_agent_and_controller/gitpodURL.sh && ./03_maven_tutorial/gitpodURL.sh && ./gitpodURL.sh
-
-ports:
-  - name: Jenkins controller
-    port: 8080
-    protocol: http
-    onOpen: open-browser
-    description: The Jenkins Controller to use for the tutorials
```

Comment:
Why removing this part?

Reply:
It is not working as intended for some reason. and the popup(that lead to Jenkins) that used to come is also not coming because of this

Reply:
Got it, thanks 👍
We can merge as is, then.
Could you please create another issue describing what you just wrote, so we don't forget it?